test/testdata/rbi/csv.rb

```diff
@@ -8,10 +8,10 @@
 end
 
 csv = CSV::Table.new([CSV::Row.new(['1', '2'], [1, 2]), CSV::Row.new(['1', '2'], [2, 3])])
-T.assert_type!(csv, CSV::Table[T.any(CSV::Row, T::Array[T.nilable(BasicObject)])])
```

Comment:
This created an error at runtime: `NoMethodError: undefined method '[]' for CSV::Table:Class`.

```

```